packages/portal/src/components/item/ItemLanguageSelector.vue

```diff
@@ -46,6 +45,12 @@
                   </b-dropdown-item>
                 </b-dropdown>
               </i18n>
+              <b-button
```

Comment:
Why is this a button? Clicking it does not do anything.

Reply:
The bootstrap-vue docs examples use buttons for the tooltip trigger elements, see https://bootstrap-vue.org/docs/components/tooltip
It might not have to be a button, but it should be an element that can receive focus, so tooltips also are shown on focus (for example keyboard only navigation).

Reply:
Understood, thank you.